giddy/directional.py

```diff
@@ -349,10 +350,10 @@ def plot_origin(self):  # TODO add attribute option to color vectors
         Plot vectors of positional transition of LISA values starting
         from the same origin.
         """
-        import matplotlib.cm as cm
+        # import matplotlib.cm as cm
```

Comment:
For all commented out lines that I added, they were raising linting errors in `ruff`.

Reply:
I see. I think these lines are redundant and we should simply remove them.

Reply:
So these aren't actually redundant, it's just that `import matplotlib.cm as cm` *isn't used*. That goes for all other commented out lines in this PR (except for L284-289 in [`markov.py`](https://github.com/pysal/giddy/pull/202/files#r1347668016)).